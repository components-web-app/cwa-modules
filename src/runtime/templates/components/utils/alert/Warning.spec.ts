// @vitest-environment happy-dom
import { describe, expect, test } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Warning from './Warning.vue'

function createWrapper (slot: string) {
  return shallowMount(Warning, {
    slots: {
      default: slot
    }
  })
}

describe('Warning', () => {
  describe('snapshot', () => {
    test('should match snapshot with passed slot content', () => {
      const wrapper = createWrapper('<div>This is test</div>')

      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
