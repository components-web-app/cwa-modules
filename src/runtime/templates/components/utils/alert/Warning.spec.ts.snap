// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Warning > snapshot > should match snapshot with passed slot content 1`] = `
<div
  class="cwa-flex cwa-items-start cwa-bg-red-500 cwa-text-white cwa-text-sm cwa-font-bold cwa-px-4 cwa-py-3"
  role="alert"
>
  
  <div>
    This is test
  </div>
  
</div>
`;
